migrate/20230725_billing_meters.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    create_table(:billing_meter) do
+      column :id, :uuid, primary_key: true, default: nil
+      column :resource_type, :text, collate: '"C"', null: false
+      column :resouce_family, :text, collate: '"C"', null: false
+      column :location, :text, collate: '"C"', null: false
+      column :unit_price, :numeric, null: false
+      index [:resource_type, :resouce_family, :location], unique: true
+    end
+
+    # These are valid UBIDs for TYPE_BILLING_METER
```

Comment:
I think it should be okay to require in the UBID code into migrations, though we should avoid dependencies on models per normal.

Reply:
Not very happy to put these here, but these ids needs to be generated offline and somehow put into to database, so migrations seemed like an OK place to do that.

Reply:
Well... they don't need to be valid UBIDs I guess.

Reply:
I decided to keep them valid UBIDs for now. If we want, ignoring their UBIDness is easy but making non UBID uuids to UBID is more difficult.